tools/build.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+const commandRunner = require('./run-command');
+
+const build = 'npm run build';
+const build_db = 'npm run build -- --public-url=/"$GH-PAGES/"';
+
+var mkdirp = require('mkdirp');
+var rimraf = require('rimraf');
+var copy = require('copy');
+
+rimraf('./dist', {}, function() {});
+
+mkdirp('dist', function(err) {
+    if (err) console.error(err);
+    else console.log('Created dist directory');
+});
+
+commandRunner(build, 'src/docs/website')
+    .then(() =>
+        copy('./src/docs/website/build/Next/**', './dist', function(err, file) {
+            if (err) console.error(err);
+        })
+    )
+    .then(() =>
+        mkdirp('dist/editor', function(err) {
+            if (err) console.error(err);
+            else console.log('Created dist/editor directory');
+        })
+    )
+    .then(() => commandRunner(build, 'src/editor'))
+    .then(() =>
+        copy('./src/editor/dist/**', './dist/edtor', function(err, file) {
```

Comment:
Same thing as with mkdirp. We should promisify this and inject it into the promise chain.

Reply:
I can promisify both rimraf and mkdirp. To do that, would I do something similar to `run-command.js` where I pass the arguments there and then promisify it?

Reply:
Yes